src/backend/marsha/core/serializers.py

```diff
@@ -478,15 +496,22 @@ def get_url(self, obj):
         if obj.uploaded_on is None:
             return None
 
+        extension = "." + obj.extension if obj.extension else ""
```

Comment:
same as above, nicer to lstrip it below than add it.

Reply:
Here I'm sure there is no leading dot, I must add it.

Reply:
but add it directly in the string below no?

Reply:
No because if there is no extension, the string should end with `.`

Reply:
And this is also why all your comments with `lstrip` are in fact not a solution. I have to rollback them.